src/Core/SecretsManager/Models/Data/ApiKeyDetails.cs

```diff
@@ -4,14 +4,16 @@ namespace Bit.Core.SecretsManager.Models.Data;
 
 public class ApiKeyDetails : ApiKey
 {
+    public string ClientSecret { get; set; }
```

Comment:
Should we add a notice that this field is deprecated?

Reply:
resolved in https://github.com/bitwarden/server/pull/2943/commits/9c72e9e3178b0baf7c8f9e1cb7bbe2741fc4e3ea